docs/setup.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "databricks"
+page_title: "Provider: Databricks"
+sidebar_current: "docs-databricks-setup"
+description: |-
+  Databricks sync databricks.
+---
+
+# Databricks-Sync
+
+Databricks-Sync is for workspace disaster recovery. You can use it to import or export some, or all, objects within a workspace. Databricks-Sync can also be used as a migration utility.
+
+## Dependencies
+
+This project requires the following environment dependencies:
+
+* [golang 1.13.x](https://golang.org/dl/)
```

Comment:
@fartzy we should remove Go from here and in step #2 
We do not need it anymore

Reply:
Ok I took golang out